tests/src/ExistingSite/ShowcaseExistingSiteCreatePersonTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_showcase\ExistingSite;
+
+use Drupal\file\Entity\File;
+use Drupal\media\Entity\Media;
+use Drupal\Tests\media\Traits\MediaTypeCreationTrait;
+use Drupal\Tests\TestFileCreationTrait;
+
+/**
+ * Class to test Person content type on existing site tests.
+ */
+class ShowcaseExistingSiteCreatePersonTest extends ShowcaseExistingSiteTestBase {
```

Comment:
No need to put the prefix ShowcaseExistingSite, we said we would have to drop this from the other tests too.
And no need to write Create either.

```suggestion
class PersonContentTest extends ShowcaseExistingSiteTestBase {
```

Reply:
It's a good idea renaming now event and news tests  or should I create a follow up ticket to do?

Reply:
Lets create a small ticket where we harmonize the test names